File: sign/vue/src/view/cn/moonnow/sign/account/AccountQuery.js
export default class AccountQuery {
  constructor () {
    this.accountId = null // 账号编号
    this.account = null // 账号
    this.password = null // 密码
    this.mobilePhoneNumber = null // 手机号
    this.nexusType = null // 关联类型
    this.nexusObjectId = null // 对象编号
    this.versions = null // 版本号
    this.accountIdAndin = null // 账号编号Andin查询
    this.accountAndeq = null // 账号Andeq查询
    this.accountAndKeyLike = null // 账号AndKeyLike查询
    this.passwordOrKeyLike = null // 密码OrKeyLike查询
    this.mobilePhoneNumberOrKeyLike = null // 手机号OrKeyLike查询
    this.nexusTypeOrKeyLike = null // 关联类型OrKeyLike查询
    this.nexusObjectIdOrKeyLike = null // 对象编号OrKeyLike查询
  }
}

File: sign/sign-interface/src/main/java/cn/moonnow/sign/query/AccountQuery.java
package cn.moonnow.sign.query;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

/**
 * 账号 DEV_ACCOUNT 查询类
 */
@Data
public class AccountQuery implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String accountId; // 账号编号

  protected String account; // 账号

  protected String password; // 密码

  protected String mobilePhoneNumber; // 手机号

  protected String nexusType; // 关联类型

  protected String nexusObjectId; // 对象编号

  protected Integer versions; // 版本号

  protected List<String> accountIdAndin; // 账号编号Andin查询

  protected String accountAndeq; // 账号Andeq查询

  protected String accountAndKeyLike; // 账号AndKeyLike查询

  protected String passwordOrKeyLike; // 密码OrKeyLike查询

  protected String mobilePhoneNumberOrKeyLike; // 手机号OrKeyLike查询

  protected String nexusTypeOrKeyLike; // 关联类型OrKeyLike查询

  protected String nexusObjectIdOrKeyLike; // 对象编号OrKeyLike查询

}
